azure-pipelines/templates/1espackage.yml

```diff
@@ -0,0 +1,17 @@
+steps:
+  - task: Npm@1
+    displayName: "\U0001F449 Package"
+    inputs:
+      command: custom
+      customCommand: run package
+    condition: succeeded()
+
+  - task: CopyFiles@2

```

Comment:
1ES has a publish task built in, you just have to copy the files to the publish folder